src/assets/locales/en/translation.json

```diff
@@ -631,5 +632,76 @@
     "total_governance_locked": "Total {{token}} Locked",
     "available_to_stake": "Available to stake",
     "voting_power_governance": "Voting Power {{token}}"
+  },
+  "transaction": {
+    "confirm_transaction_wallet": "Confirm this transaction in your wallet",
+    "confirm_transaction": "Confirm transaction",
+    "transaction_processing": "Transaction processing",
+    "transaction_failed": "Transaction failed",
+    "transaction_successful": "Transaction successful",
+    "swapping_to": "Swapping {{fromAmount}} {{fromCurrency}} to {{toAmount}} {{toCurrency}}",
```

Comment:
@tomjeatt @nud3l could you review this transaction texts? Do you need me to write examples or are these placeholders enough ?

Reply:
Will have a look now as well